src/nvim/os/os_defs.h

```diff
@@ -18,11 +18,15 @@
 #endif
 
 #if defined(UFS_MAXNAMLEN) && !defined(MAXNAMLEN)
-# define MAXNAMLEN UFS_MAXNAMLEN    /* for dynix/ptx */
+# define MAXNAMLEN UFS_MAXNAMLEN    // for dynix/ptx
```

Comment:
If you use `//` comments, e.g.

```c
MAXNAMLEN * 3
```

will expand to

```c
UFS_MAXNAMELEN // for dynix/ptx * 3
```

and otherwise

```c
UFS_MAXNAMLEN /* for dynix/ptx */ * 3
```